### PR TITLE
fix invalid image error message and return False if not image

### PR DESCRIPTION
**Pull request by RaynorChavez**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Error message change

* **What is the current behavior?** (You can also link to an open issue here)
When an invalid image is processed, marqo gives an error: `input str of `{image_path}` is not a local file or a valid url`
which might confuse users.

* **What is the new behavior (if this is a feature change)?**
The error message is changed to:
```
input str of `{image_path}` is not a local file or a valid url. 
If you are using Marqo Cloud, please note that images can only be downloaded "
from a URL and local files are not supported. "
If you are running Marqo in a Docker container, you will need to use a Docker "
volume so that your container can access host files."
```
for clarity

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Have unit tests been run against this PR?** (Has there also been any additional testing?)

* **Related Python client changes** (link commit/PR here)

* **Related documentation changes** (link commit/PR here)

* **Other information**:

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by wanliAlex — CHANGES_REQUESTED** [review]
Can you add a test for this (ensure we return bool instead of None)? Maybe also add this link https://docs.marqo.ai/2.10/Guides/Advanced-Usage/images/ in the error message

**Review by wanliAlex — DISMISSED** [review]

File: src/marqo/s2_inference/clip_utils.py [hunk 1]
```diff
@@ -116,7 +116,13 @@ def load_image_from_path(image_path: str, image_download_headers: dict, timeout_
...
                 metrics_obj.stop(f"image_download.{image_path}")
     else:
-        raise UnidentifiedImageError(f"input str of `{image_path}` is not a local file or a valid url")
+        raise UnidentifiedImageError(f"Input str of {image_path} is not a local file or a valid url. "
+                                     f"If you are using Marqo Cloud, please note that images can only be downloaded "
+                                     f"from a URL and local files are not supported. "
+                                     f"If you are running Marqo in a Docker container, you will need to use a Docker "
+                                     f"volume so that your container can access host files. "
+                                     f"For more information, please refer to: "
+                                     f"https://docs.marqo.ai/latest/Guides/Advanced-Usage/images/")
```
Comment (farshidz) [inline src/marqo/s2_inference/clip_utils.py]: Use `marqo_docs` module so link it tested and has the right version

File: tests/s2_inference/test_clip_utils.py [hunk 2]
```diff
@@ -56,6 +56,43 @@ def test_downloadImageFromRrlCloseCalled(self, MockCurl):
         # Check if c.close() was called
         mock_curl_instance.close.assert_called_once()
 
+    def test_is_image(self):
+        # Test with valid image file extensions
+        self.assertTrue(clip_utils._is_image('image.jpg'))
```
Comment (farshidz) [inline tests/s2_inference/test_clip_utils.py]: Use subtests so in one run we know which ones passed and which ones failed

**Review by farshidz — CHANGES_REQUESTED** [review]

File: src/marqo/s2_inference/clip_utils.py (hunk 1 above)
Comment (RaynorChavez) [reply src/marqo/s2_inference/clip_utils.py]: addressed

File: tests/s2_inference/test_clip_utils.py (hunk 2 above)
Comment (RaynorChavez) [reply tests/s2_inference/test_clip_utils.py]: addressed

**Review by farshidz — APPROVED** [review]
